src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  box-sizing: border-box;
```

Comment:
move it to `*`

Reply:
If i'm transfering all this 4 options to * - everything breaking in the code

Reply:
move just box sizing
and remove it everywhere